server/index.js

```diff
@@ -26,13 +26,13 @@ const allProducts = `
          "image",
          "shortDescription"
     from "products"
-`;
+`; // using sql to get the products
```

Comment:
Good work! The only thing I'd recommend is moving your allProducts const inside of your products app.get, since that is the one place you'll be using it.

Reply:
Ah thank you! Sent an update.